qiita_ware/ebi.py

```diff
@@ -272,8 +271,8 @@ def generate_study_xml(self):
 
         Returns
         -------
-        str
-            string with study XML values
+        Element
+            Object with study XML values
```

Comment:
The reason we use `np` and `pd` instead of `numpy` and `pandas` is because `np` and `pd` are standard abbreviations. Is `ET` a standard abbreviation of `xml.etree.ElementTree`?